apps/estuary/test/integration/estuary_test.exs

```diff
@@ -0,0 +1,119 @@
+defmodule Estuary.EstuaryTest do
+  use ExUnit.Case
+  use Placebo
+  use Divo
+  import SmartCity.TestHelper, only: [eventually: 1]
+
+  @elsa_endpoint Application.get_env(:estuary, :elsa_endpoint)
+  @event_stream_topic Application.get_env(:estuary, :event_stream_topic)
+  @event_stream_table_name Application.get_env(:estuary, :event_stream_table_name)
+
+  test "Topic is created when Estuary starts" do
+    assert Elsa.Topic.exists?(@elsa_endpoint, @event_stream_topic)
+  end
+
+  test "should create event_stream table when estuary starts" do
+    expected_table_value = [[@event_stream_table_name]]
+
+    actual_table_value =
+      "SHOW TABLES LIKE '#{@event_stream_table_name}'"
+      |> Prestige.execute()
+      |> Prestige.prefetch()
+
+    assert expected_table_value == actual_table_value
+  end
+
+  test "should check all the columns exists in the table when estuary starts" do
```

Comment:
I'd combine this with  the test above. The test above is checking that the table exists and may as well also check the table has the proper columns.